src/components/ha-language-picker.ts

```diff
@@ -144,6 +146,7 @@ export class HaLanguagePicker extends LitElement {
         @closed=${stopPropagation}
         fixedMenuPosition
         naturalMenuWidth
+        ?inlineArrow=${this.inlineArrow}
```

Comment:
```suggestion
        .inlineArrow=${this.inlineArrow}
```

Reply:
Thanks, I read again the documentation how to use `?` :+1: